_docs/style/text-conventions/capitalization.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Capitalization
+layout: docs
+parent: Text Conventions
+category: Style
+usage: |
+  User interface (UI) elements use a combination of title-style, sentence-style, and all-caps capitalization. When you use different capitalization styles for different types of content, you can improve scannability, organize information, and guide users to key actions. It is important to apply these capitalization guidelines consistently to make the UI clear and succinct.
+resource: false
+status: complete
+last-modified: 2018-06-28
+---
+
+{% include toc.html %}
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## Title-style capitalization
+
+For title-style capitalization, capitalize the first letter of nouns, adjectives, verbs, adverbs, and pronouns in the text. You also capitalize the first and last words of the text. However, use lowercase for the following exceptions:
+
+- Articles (*a, an, the*) unless they are the first or last words of the element.
+- Coordinating conjunctions (*and, or*).
+- Prepositions with fewer than five letters (*to, on, at, by, from*) unless they are the first or last words of the element.
+- Infinitives (*to, in, an*).
+- Capitalize the second word in a hyphenated compound if it would normally be capitalized as a single word. For example, you would capitalize both words in **Quick-Find** Options, but you would only capitalize **Add** in Add-on Choices.
+
+The following table includes examples for title-style capitalization:
+
+{% figure [caption:"Title-style capitalization examples"] [class:"image bg-light border"] %}
+| <hx-icon type="checkmark" class="good-idea"></hx-icon> Use | <hx-icon type="times" class="bad-idea"></hx-icon> Don't Use  |
+| --- | --- |
+| Restore Defaults | Restore defaults |
+| Allocate to Project | Allocate to project |
+| Show Add-on in Menu | Show Add-On in Menu |
+{: class="hxTable"}
+{% endfigure %}
+
+The following elements use title-style capitalization:
+
+- Button labels
+- Chart titles
+- Checkbox and radio button field names
+- Icon labels
+- Links used for an action or navigation
+- Menu items (both menu bar and context menus)
+- Menu titles
+- Modal titles
+- Page titles
+- Section headings
+- Step titles
+- Table column headers
```

Comment:
Currently, table headers are ALL CAPS. 
They might not be forever, but for the moment they are. move Table headers to the list in the All-Caps section.

Reply:
i see they're in all caps already, so just remove from this section and we're gtg